index.bs

```diff
@@ -267,6 +267,52 @@ partial interface NeuralNetworkContext {
     The element-wise binary addition will be broadcasted according to [[!numpy-broadcasting-rule]]. The rank of the output tensor is the maximum rank of the input tensors. For each dimension of the output tensor, its size is the maximum size along that dimension of the input tensors.
 </div>
 
+### batchNormalization ### {#api-neuralnetworkcontext-batchnorm}
+Normalize the tensor values across dimensions in a batch through a specialized [transform](https://en.wikipedia.org/wiki/Batch_normalization#Batch_Normalizing_Transform) introduced in this [paper](https://arxiv.org/abs/1502.03167). The *mean* and *variance* tensors are previously calculated during model training pass.
+<script type=idl>
+partial interface NeuralNetworkContext {
+  Operand batchNormalization(Operand input, Operand mean, Operand variance, Operand scale, 
+                             Operand bias, optional long axis = 1, optional float epsilon = 1e-5);
+};
+</script>
+<div algorithm=batchnorm>
+    **Arguments:**
+        - *input*: an {{Operand}}. The input N-D tensor.
+        - *mean*: an {{Operand}}. The 1-D tensor of the batch mean values.
+        - *variance*: an {{Operand}}. The 1-D tensor of the batch variance values.
+        - *scale*: an {{Operand}}. The 1-D tensor of the scaling values.
+        - *bias*: an {{Operand}}. The 1-D tensor of the bias values.
+        - *axis*: an optional {{long}}. The index for the feature dimension of the input shape for which 
+            the normalizing batch is defined and that the mean and variance values are computed. 
+            When it's not specified, the default value is 1.
+        - *epsilon*: an optional {{long}}. The Epsilon value to prevent computational error due to divide-by-zero.
```

Comment:
It should be an optional {{float}}.

Reply:
@huningxin Please take a look at the new commit.